include/libdnf5/rpm/nevra.hpp

```diff
@@ -65,6 +65,15 @@ struct LIBDNF_API Nevra {
     /// @return `true` if all Nevra attributes (`name`, `epoch`, `version`, `release` and `arch`) match.
     bool operator==(const Nevra & other) const;
 
+    /// Compares Nevra attributes in order: `name`, `epoch`, `version`, `release`, `arch`, and checks
+    /// if any is less than the corresponding attribute.
+    /// @return `true` if any of the attributes is less than the corresponding one. Return `false` otherwise.
+    bool operator<(const Nevra & other) const;
```

Comment:
This patch adds new functions to the public API. That means this patch needs increasing DNF5 version from  5.2.12.0 to  5.2.13.0. See VERSION.cmake file.

Reply:
Could you please reverse the commit order, or squash them into a single commit? I try to avoid a situation where one gets a checkout with the new operators, but without the bumped version.

Reply:
Done. I've also updated the `spec` file, but skipped the changelog section. I guess there should be github-action bot or smth like that to do it properly, or ?

Reply:
Yeah, thanks for the spec file. But still could you swap the two commits?

Reply:
It's just 1 now